docs/presets/README.md

```diff
@@ -466,3 +466,60 @@ format = 'via [$symbol]($style)'
 [zig]
 format = 'via [$symbol]($style)'
 ```
+
+## Pure
+
+This preset emulates the look and behavior of [Pure](https://github.com/sindresorhus/pure).
+
+![Screenshot of Pure preset](/presets/pure-prompt.png)
+
+### Configuration
+
+```toml
+format = """
+$username\
+$hostname\
+$directory\
+$git_branch\
+$git_state\
+$git_status\
+$cmd_duration\
+$line_break\
+$python\
+$character"""
+
+[directory]
+style = "blue"
+
+[character]
+success_symbol = "[❯](5)"
+error_symbol = "[❯](red)"
+vicmd_symbol = "[❮](green)"
+
+[git_branch]
+format = "[$branch]($style)"
+style = "242"
```

Comment:
It seems `242` is fairly similar to what most themes assign to `bright-black` (which is `8`), as shown here:
![image](https://user-images.githubusercontent.com/4658208/135533766-a05cb6ae-56a4-4d73-abc8-ea84e47a1aa5.png)

Would you be open to having it changed for the values set to `242`?
It would allow for the preset to better match themes with different color schemes.

```suggestion
style = "bright-black"
```